src/w3d/renderer/assetmgr.cpp

```diff
@@ -746,13 +749,62 @@ RenderObjClass *GameAssetManager::Create_Render_Obj(
     return robj;
 }
 
+bool Get_Mesh_Color_Methods(MeshClass *mesh, bool *housecolor, bool *zhc)

```

Comment:
Arguments are all pointers, not references?

Reply:
I have no evidence to be sure but it looks that way at least.